icpc-wf/ansible/files/grafana/my.cnf.j2

```diff
@@ -1,6 +1,6 @@
 [client]
 user = collectdstatus
 password = {{ COLLECTD_MYSQL_PASSWORD }}
-host = {{ hostvars[host].ansible_host }}
+host = 127.0.0.1
```

Comment:
Was this change intentional?

Reply:
Yes, because we have 2 SQL servers we need to scrape from localhost and then export, instead of crawl from the monitoring server and then send to grafana on localhost.